test-monitor/nitestmonitor.yml

```diff
@@ -1383,6 +1437,40 @@ paths:
           $ref: '#/responses/Unauthorized'
         default:
           $ref: '#/responses/Error'
+  /v2/query-steps:
+    post:
+      tags: [steps]
+      summary: Advanced Test Step Query
+      description: Uses the Dynamic Linq query language to specify a filter and return test steps. An empty request body queries all test steps.
+      operationId: query-steps-v2
+      x-ni-operation: querySteps
+      x-ni-privilege: Read
+      parameters:
+        - in: body
+          name: postBody
+          description: Query filter
+          required: false
+          schema:
+            $ref: '#/definitions/AdvancedQuery'
+        - in: query
+          name: skip
+          type: integer
+          format: int32
+          default: 0
+          minimum: 0
+        - in: query
+          name: take
+          type: integer
+          format: int32
+          default: -1
+          minimum: -1
```

Comment:
Same: Both skip and take need a description. I would also suggest changing take to be have valid values of "null" and then integers >=0, much like this API (which was based on a suggestion from Paul):
https://github.com/ni/systemlink-OpenAPI-documents/blob/master/user-data/niuserdata.yml#L792